server/service/mongo/kv/kv_service.go

```diff
@@ -74,7 +75,7 @@ func (s *Service) CreateOrUpdate(ctx context.Context, kv *model.KVDoc) (*model.K
 		}
 
 	}
-	kv.LabelID = labelID.Hex()
+	kv.LabelID = string(labelID)
```

Comment:
Why? The value type is not changed.

Reply:
labelID's type is id.ID

kv.LabeldID's type is string

need to convet it